modules/dynamic-operator/config/manager/manager.yaml

```diff
@@ -178,6 +188,11 @@ spec:
                 secretKeyRef:
                   name: kuberlogic-config
                   key: SENTRY_DSN
+            - name: KUBERLOGIC_DEPLOYMENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: kuberlogic-config
+                  key: DEPLOYMENT_ID
```

Comment:
How the DEPLOYMENT_ID is generated?

Reply:
It's generated only during first install, here:
https://github.com/kuberlogic/kuberlogic/blob/KL-343/modules/dynamic-apiserver/pkg/cli/install_operation.go#L118